src/App.js

```diff
@@ -1,95 +1,91 @@
-import './App.scss';
-import './assets/styles/scss/abstracts/variables.scss'
 import { useState } from 'react';
+import './App.scss';
+import './assets/styles/scss/abstracts/variables.scss';
+import PaintingInfoOverlay from './components/2D/PaintingInfoOverlay';
+import UI from './components/2D/UI';
+import UploadJSON from './components/2D/UploadJSON';
+import imageFromPainting from './utils/imageFromPainting';
+import removeTextInBrackets from './utils/removeTextInBrackets';
+import World from './components/3D/World';
 
 function App() {
-
- const [bestOfs, setBestOfs] = useState(null)
- const [loading, setLoading] = useState(false)
-
- const onFileChange = event => { 
-
-  setLoading(true)
-
-  var files = event.target.files
-
-  if (files.length <= 0) {
-    return false;
-  }
-
-  files[0].text().then(function(text) { 
-    var result = JSON.parse(text);
-    const filteredItems = result.items.filter(item => item.isBestOf === true)
-    const filteredItemsSorted = filteredItems.sort((a,b) => (a.sortingNumber > b.sortingNumber) ? 1 : ((b.sortingNumber > a.sortingNumber) ? -1 : 0))
-    setBestOfs(filteredItemsSorted)
-    console.log(filteredItemsSorted)
-    setLoading(false)
-  })
-
-}; 
-
-const removeTextInBrackets = (text) => {
-
-  const roundBracketsRemoved = text.split('(')[0]
-
-  const squareBracketsRemoved = roundBracketsRemoved.split('[')[0]
-
-return squareBracketsRemoved
-}
+  const [bestOfs, setBestOfs] = useState(null);
+  const [loading, setLoading] = useState(false);
+  const [focusedPainting, setFocusedPainting] = useState(null);
+  const stepSize = 4;
+
+  /**
+   *
+   * @returns Object containing all Infomation displayed in the Info Overlay
+   */
+  const paintingInfo = () => {
```

Comment:
I like that you code is very organized, commented and functions are being outsourced in a logical and suffisient way. Also using a utils folder is beautifully structured